src/Data/Dungeons/DungeonRepository.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections.Generic;
+using System.Linq;
+using System;
+
+namespace ThirdRun.Data.Dungeons
+{
+    /// <summary>
+    /// Repository containing all predefined dungeon definitions with their level ranges and map configurations
+    /// </summary>
+    public static class DungeonRepository
+    {
+        private static readonly List<Dungeon> Dungeons = new List<Dungeon>
+        {
+            new Dungeon("Caverne des Gobelins", 1, 3, new List<DungeonMapDefinition>
+            {
+                new DungeonMapDefinition("Entrée de la caverne avec quelques gobelins", false, 3, 1, 2),
```

Comment:
Dungeon repository should describe the exact combination of type of monsters that are present, for example 4 goblins and 2 spiders in the goblin caves.